polaris.shopify.com/content/foundations/design/data-visualizations.md

```diff
@@ -12,10 +12,7 @@ single, specific question.
 This section outlines data visualization practices at Shopify and how to
 leverage them.
 
-<div class="Disclaimer">
-<span>Note</span>
-We’re working on updates to the data visualization principles and rules. Stay tuned!
-</div>
+**Note:** We’re working on updates to the data visualization principles and rules. Stay tuned!
```

Comment:
Not this PRs job, but I think we should remove this. Promises of future updates typically just leads to empty promises being strewn all of the site.

Reply:
I will remove it now. Good point.